### ISSUE TITLE
Population response with containerGeometry is a wrong value

### ISSUE THREAD

**Issue by ajturner** [uscensusbureau/citysdk#91]
the [HUD Demo](http://esridc.github.io/citysdk-hud/) always returns `population: 1130` regardless of the area selected

![citysdk-hud](https://cloud.githubusercontent.com/assets/1218/7717886/ba9aa324-fe76-11e4-852e-6318ab184b74.png)

/cc @afrasier-bah

**Comment by afrasier-bah** [comment]
Thanks, the containerGeometry functionality needs a little bit of TLC. I figured you'd be the one to break the most stuff ;)

**Comment by ajturner** [comment]
> I figured you'd be the one to break the most stuff ;)

I'll take that as a compliment! :rocket:

**Comment by afrasier** [comment]
This will come out with the new code sometime at the end of this week (along with more extensive documentation) - but I found the bug that was responsible for this. Additionally, I improved this feature such that

* If you're requesting via a containerGeometry, you no longer need to specify a location (i.e. lat/lng, state, zip, or address)
* There are two new "container" types:
     * 'geometry' - Returns all geometry that intersects or is within the boundaries of the containerGeometry
     * 'geometry_within' - Returns all geometry that is entirely contained within the boundaries of containerGeometry

Here's an example request from the demo for this feature that will go up at the end of the week:
```
 var request = {
                variables: [
                    "age",
                    "population",
                    "income"
                ],

                level: "tract",
                sublevel: true,
                container: "geometry",
                containerGeometry: census.GEOtoESRI(triangleGeojson)[0].geometry
            };
```

Where triangleGeojson is some geojson. We grab the geometry from the first returned geometry object from Terraformer. I'm not sure if it's better to just do that for the user inside of GEOtoESRI, but I'm erring on the side of giving the user the entire return to do with as they will.

**Comment by ajturner** [comment]
Was the bug fix deployed? This will affect the webcast on Monday. The demo is linked to master, so if it's pushed it should work.

**Comment by afrasier** [comment]
Resolved f1cabdf45be7797e9b3266c39a6b0db60db7549e but keep your eyes peeled for documentation regarding the new container types - though my comment before should suffice for now :)
